example/ios/example/Info.plist

```diff
@@ -5,7 +5,7 @@
 	<key>CFBundleDevelopmentRegion</key>
 	<string>en</string>
 	<key>CFBundleDisplayName</key>
-	<string>Dev-RN-PhylloSampleApp</string>
+	<string>Dev RN SampleApp</string>
```

Comment:
Any reason to change this?

Reply:
@akshaybhalotia no reason , give me one specific example name for this . itsn't able to show full name when we install example app in iphone so make it short .